### PR TITLE
Pr add role docker

### PR DESCRIPTION
**Pull request by carneirofc**

### CODE REVIEW

File: playbook-servers-docker.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      file:
+        path: "/etc/docker/certs.d/{{ docker_registry_server }}/"
+        state: directory
+        mode: "0755"
+
+    - name: Copy self signed crt to the client machine
+      ansible.posix.synchronize:
+        src: "{{ docker_registry_ssl_crt_path }}"
+        dest: "/etc/docker/certs.d/{{ docker_registry_server }}/ca.crt"
+      delegate_to: "{{ docker_registry_server }}"
```
Comment (lerwys) [inline playbook-servers-docker.yml:30]: - Do we need to also put the certificate at `usr/local/share/ca-certificates/{{ docker_registry_server }}.crt`?
- I think we might need to issue a `update-ca-certificates` command. Or is this step not needed?
- I think we need to restart docker daemon service after changing its certificate

**Review by lerwys — CHANGES_REQUESTED** [review]
Some minor comments:

- Should we fix the ansible lint warnings in third-party roles? I'm not sure.
- I think we should add a `requirements.yml` file at the root of this repo with the following:

```yaml
---
collections:
  - name: ansible.posix
```
And maybe add a section `Dependencies` on `README.md` mentioning this and how to install it. Something along the lines of:

```
ansible-galaxy install -r requirements.yml
```
- Add a target in Makefile for compliance?

File: playbook-servers-docker.yml (hunk 1 above)
Comment (carneirofc) [reply playbook-servers-docker.yml:30]: Just copying the content to the corresponding folder is enough. The docker service will automatically detect the certificate.

**Comment by lerwys** [comment]
Seems good to me

**Review by lerwys — APPROVED** [review]

**Comment by lerwys** [comment]
@xresende Can you just give a quick look in that PR please?

**Pull request merged**